README.md

```diff
@@ -179,7 +251,11 @@ this script.
 
 ```shell
 cd <CSDK_ROOT>
-python3 docs/doxygen/generate_docs.py --root .
+python3 tools/doxygen/generate_docs.py --root .
 ```
 
-The generated documentation landing page is located at **docs/doxygen/output/html/index.html**.
```

Comment:
Was this line supposed to have been deleted? It seems like it's part of the documentation section above